PVCs with StorageOS scenario (#34)
storageos/tutorials: pvc-basic/040-replication-in-pvc.md

> It is often needed to define feature labels for each PVC instead of using the

Reviewer: ```suggestion
It can be useful to define feature labels for each PVC instead of using the
```
